src/main/java/uk/gov/hmcts/reform/profilesync/service/impl/ProfileSyncServiceImpl.java

```diff
@@ -131,7 +133,8 @@ public String getS2sToken() {
     public void updateUserProfileFeed(String searchQuery) throws UserProfileSyncException {
         log.info("Inside updateUserProfileFeed");
         String bearerToken = BEARER + getBearerToken();
```

Comment:
Please avoid + use  concat.

Reply:
Thanks Abhijit. I have now used concat()for prod code.